provider/internal/java/dependency.go

```diff
@@ -258,9 +258,14 @@ func (p *javaServiceClient) parseDepString(dep, localRepoPath string) (provider.
 	fp := filepath.Join(localRepoPath, strings.Replace(parts[0], ".", "/", -1), parts[1], d.Version, fmt.Sprintf("%v-%v.jar.sha1", parts[1], d.Version))
 	b, err := os.ReadFile(fp)
 	if err != nil {
-		return d, err
+		// Log the error and continue with the next dependency.
+		p.log.V(7).Info("Error reading SHA hash file for dependency %s: %s\n", dep, err)
```

Comment:
```suggestion
		p.log.V(5).Error(err, "error reading SHA hash file for dependency", "dep", d.Name)
```

Reply:
oh my bad .. thanks for the suggestion. will update now